java-checks/src/test/files/checks/PrimitiveWrappersInTernaryOperatorCheck.java

```diff
@@ -0,0 +1,15 @@
+class A {
+  void foo() {
```

Comment:
Isn't this missing a test case with a primitive ? 
int i;
long l;
true ? i : l; //Compliant ?